frontend/src/scenes/insights/FunnelInsight.tsx

```diff
@@ -26,6 +26,7 @@ export function FunnelInsight(): JSX.Element {
                 onSaveCohort={() => {
                     setCohortModalVisible(true)
                 }}
+                showModalActions={!filters.aggregation_group_type_index}
```

Comment:
This introduces a bug - `0` is falsy.

Use `!= undefined` as elsewhere in the code.